packages/react-api/src/types.d.ts

```diff
@@ -11,13 +11,13 @@ interface CredentialsCarto {
   serverUrlTemplate?: string
 }
 
-interface CredentialsCartoCloudNative {
```

Comment:
After talking to @borja-munoz, it looks like these names should be changed, so commiting a small rename now